api/src/main/java/uk/gov/dhsc/htbhf/dwp/service/EligibilityService.java

```diff
@@ -43,12 +47,16 @@ public EligibilityResponse checkEligibility(EligibilityRequest eligibilityReques
         String nino = eligibilityRequest.getPerson().getNino();
         Optional<UCHousehold> ucHousehold = ucHouseholdRepository.findHouseholdByAdultWithNino(nino);
         if (ucHousehold.isPresent()) {
-            return createEligibilityResponse(ucHousehold.get());
+            return householdVerifier.detailsMatch(ucHousehold.get(), eligibilityRequest.getPerson())
```

Comment:
This and the next ternary feel a bit too big, so I'll refactor them out into a method for clarity.

Reply:
Done.